jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/CreateFunctionDialog.kt

```diff
@@ -94,13 +99,23 @@ class CreateFunctionDialog(private val project: Project, private val initialRunt
                 title = message("lambda.service_name"),
                 content = message("lambda.function.created.notification", functionName)
             )
-            LambdaTelemetry.editFunction(project, update = false, result = Result.Succeeded)
+            LambdaTelemetry.editFunction(
+                project,
+                lambdaPackageType = LambdaPackageType.from(view.configSettings.packageType().toString()),
+                update = false,
```

Comment:
Was this always a bug? Shouldnt this be an update?

Reply:
Ah yeah this was incorrect, fixed

Reply:
wait nevermind this is create, so no

Reply:
Should create be deployFunction then?

Seems odd that create == edit?